Add modules to /v2/account

ChalyFlavour opened the pull request:
Extended information about the game content available for this account. Possible flag should be at least:
gw2_base (regular GuildWars2 game, e.g. bought at guildwars2.com)
gw2_f2p (GuildWars2 f2p edition)
gw2_hot (activated GuildWars2 Heart Of Thorns addon)

Some extended flags may apply, e.g.
gw2_wvw (requirements met for playing on world-vs-world maps)
gw2_guildbank (access to guildbanks are granted)

lye commented:
Makes sense. Will probably fiddle with the enumeration values (we're mostly using `CamelCase` everywhere else) and stick 'em on a `flags` or `features` property instead of `modules`.

ChalyFlavour commented:
I corrected the account.js to have a more applicable example.
Changed "modules" to "features" and replaced examples with CamelCase to meet the guidelines.More adapted identifiers:
GW2Base (regular GuildWars2 game, e.g. bought at guildwars2.com)
GW2F2P (GuildWars2 F2P edition)
GW2HoT (activated GuildWars2 Heart Of Thorns addon)
GW2WvWAccess (requirements met for playing on world-vs-world maps)
GW2GuildBankAccess (access to guildbanks are granted)

lye commented:
I integrated a subset of your changes into a separate pull request (which are the bits I've got working).

I think the only bit that's feasible to expose for now is the game access levels -- the WvW entry rules and guild bank access are based off a lot of runtime-configurable factors (internally) which make exposing their state exceptionally convoluted.

I've also adjust the enumeration values to not be prefixed with `GW2` since the game is implied.
